xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -366,9 +528,19 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a value that indicates which value property <c>Value</c> property to use to get the token value.</summary>
+        <value><see langword="true" /> if <see cref="P:System.Text.Json.Utf8JsonReader.ValueSequence" /> should be used to get the token value. <see langword="false" /> if <see cref="P:System.Text.Json.Utf8JsonReader.ValueSpan" /> should be used instead.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+
+## Remarks
+
+If `HasValueSequence` is `false`, <xref:System.Text.Json.Utf8JsonReader.ValueSequence> contains useless data, likely for a previous multi-segment token. Therefore, read the token value using the <xref:System.Text.Json.Utf8JsonReader.ValueSpan> property.
```

Comment:
This PR (https://github.com/dotnet/corefx/pull/34667/) changed it so `ValueSequence` is empty (set to default) when `HasValueSequence` is false. It won't contain arbitrary previous token. We should update the docs accordingly.